GurshanBrar.md

```diff
@@ -0,0 +1,39 @@
+---
+name: "@GurshanBrar"
+project: "3d Printed Prosthetic Hand"
+---
+
+# Project Name
+3D Printed Prosthetic Hand
```

Comment:
Hello! So our inspiration behind building the project is that we wanted to 1) experiment with the impulse sensors that provide input to the hand, and 2) help create a lightweight, cheap solution for people with disabilities.  Since it's 3d printed, its very cheap and its also not too heavy. Additionally, the fishing line provides appropriate tension so that the hand can grip into any position, like a bike handle for example.

As for our guide, we'll be watching YouTube videos about 3d printed prosthetic hands to start off. Then we'll model and test out our own variation to make the best solution. For the arduino and wiring, we'll find arduino tutorials on how to control servos, and how to receive inputs through the impulse sensors.